troubleshooting_101.rst

```diff
@@ -59,8 +59,92 @@ Walk through of a diagnosis
 Recent changes
 --------------
 
-Often problems can be traced back to recent changes.
-Problems that start around the time of a change aren't usually coincidence.
+As Roy Rapoport states, "change is often the enemey of stability." Frequently,
+you will find yourself troubleshooting outages or system issues that were
+self-inflicted.
+
+It's probably fairly obvious but when a system problem occurs or performance
+degrades, something has changed! If nothing had changed, no one would be
+complaining. Often problems can be traced back to recent changes. You
+will quickly learn that problems that start around the time of a change,
+are not usually a coincidence.
+
+As you troubleshoot an issue, you will find yourself logging in to hosts and
+monitoring systems to identify what's different. After all, the system is
+responding differently (suddenly) and it's your job to find out what's changed.
+Did someone on the team make a change? Or, was the change a degredation in the
+operating environment? Such as a service failing, a system failing, a bad disk
+or memory that's failed.
+
+Here are a few questions you may ask your servers, yourself, your teammates
+or developers who may deploy to assist in identifying a change:
+
+Who was last logged in to the system, and when?
+
+.. code-block:: console
+
+    user@opsschool ~$ last
+    brian    pts/0        nosferatu.krs.io Sun Jan  4 00:27   still logged in
+    brian    pts/0        nosferatu.krs.io Fri Jan  2 18:04 - 20:53  (02:48)
+    sjackson pts/0        192.168.10.2     Fri Jan  2 17:36 - 17:36  (00:00)
+    sjackson pts/0        192.168.10.2     Fri Jan  2 09:09 - 09:09  (00:00)
+
+What did they do?
+
+.. code-block:: console
+
+    user@opsschool ~$ sudo less /home/sjackson/.bash_history
+    ..
+    vi /etc/crontab
+    vi /etc/important-app/config
+    cd /opt/important-app
+    git checkout v1.6.0
+
```

Comment:
There may be ethical concerns around looking at a users shell history. Is this a good place to mention them? Or link to them?
Of course, when things are broken you need to have access to resources like this. But we also don't want to make it sound like people can/should look at user shell histories whenever they want to (or even as a first resort).

I don't have a good answer to this, but wanted to note this down in case others have strong feelings.